src/cmd/root.go

```diff
@@ -87,7 +87,7 @@ func setupLogging() {
 	zerolog.TimeFieldFormat = zerolog.TimeFormatUnix
 
 	if logFormat == "text" {
-		output := zerolog.ConsoleWriter{Out: os.Stderr}
+		output := zerolog.ConsoleWriter{Out: os.Stderr, TimeFormat: time.RFC3339}
```

Comment:
should we do this in our other libraries where we use zerologger?

could this been move to opslevel_common so we have a shared method for setting up the logging system?

Reply:
@rocktavious actually I'm going to revert this change and the changie file since I honestly only added it for people using `--resync-minutes`. It's not really beneficial otherwise.

I don't think it should be included in opslevel-common because it's application specific. We shouldn't bind anything logging related and let the apps do that themselves even if results in repeated code IMO.